vignettes/data_protocol.Rmd

```diff
@@ -76,15 +69,27 @@ The distributions currently included in the literature search for each pathogen/
+1. **Adding papers**: the papers collected in the literature search can be supplemented by papers known to be relevant or through being cited by a paper in the literature search. Papers may be recommended by experts in research or public health communities. We plan to use two methods of community engagement. Firstly a open-access Google sheet allows people to add their distribution data which will then be reviewed by one of the `{epiparameter}` maintainers and incorporated if it meets quality checks. The second method - not yet implemented - involves community members uploading their data to [zenodo](https://zenodo.org/), which can then be read and loaded into R using `{epiparameter}` once checked. 
 
-1. **Language restrictions**: papers in English or Spanish are currently supported in `epiparameter`. Papers written in another language but verified by an expert can also be included in the database. However, these are not evaluated with the review process described below and as a result are flagged to the user when loaded in `epiparameter`.
+1. **Language restrictions**: papers in English or Spanish are currently supported in `{epiparameter}`. Papers written in another language but verified by an expert can also be included in the database. However, these are not evaluated with the review process described below and as a result are flagged to the user when loaded in `{epiparameter}`.
 
 ## Guide to data refinement once sources identified
 
-1. **Removing duplicates**: studies that use multiple data sources should be screened against other papers included to ensure the the same data sets are not being included multiple times in `epiparameter`. This is done by checking whether papers are the primary source of data or are retrieving and citing from another primary source.
+1. **Removing duplicates**: studies that use multiple data sources should be screened against other papers included to ensure the the same data sets are not being included multiple times in `{epiparameter}`. This is done by checking whether papers are the primary source of data or are retrieving and citing from another primary source.
 
 1. **Abstract and methods screening**: once a number of unique sources have been identified, each should be reviewed for its suitability by reviewing the abstract and searching for words or phrases in the paper that indicate it estimated the parameters of a distribution, this can include searching the methods section for words on types of distributions (e.g. lognormal), fitting procedures (e.g. maximum likelihood or bayesian), or searching the results for parameter estimates. If no distribution is described, the paper can be considered unsuitable at this stage.
```

Comment:
so we are discarding papers without distributions in the end?

Reply:
Update to mention that we accept entries in the database that do not specify a distribution.